### PR TITLE
LPD-33971 Change how we are indexing resolution

### PR DESCRIPTION
**Pull request by Mikellorza**
https://liferay.atlassian.net/browse/LPD-33971

# What is this trying to solve?

Change the way what we are indexing resolution for this type of use case:

If the image has a resolution that includes a height that belongs to a different option than the width the image should appear in the higher option. Example: Image 600 x 288, it should appear in Medium. If the image is 350 x 700 it should also display in Medium.

# How am I fixing it?

Changing how to index resolution in order to category some image resolution in the higher category if image or video width or large is greater than the width/large high limit.

# How can I test it?

Run included tests and upload an image in document library, go to content dashboard, filter by resolution and see if the image is there.

### CODE REVIEW

**Comment by Mikellorza** [comment]
ci:test:sf

File: modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/search/spi/model/index/contributor/DLFileEntryModelDocumentContributor.java [hunk 1]
```diff
@@ -259,9 +259,19 @@ else if (tiffImageLength < tiffImageWidth) {
 						aspectRatio = "wide";
 					}
 
+					String resolution = "large";
+
+					if ((tiffImageLength <= 300) && (tiffImageWidth <= 400)) {
```
Comment (AliciaGarciaGarcia) [inline modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/search/spi/model/index/contributor/DLFileEntryModelDocumentContributor.java:264]: magic number?

maybe to extract to a constants ?

**Review by AliciaGarciaGarcia — CHANGES_REQUESTED** [review]
Failing test I seems relevant for this PR: 
```

com.liferay.document.library.internal.search.spi.model.index.contributor.test.DLFileEntryModelDocumentContributorTest > testFileEntryMetadataAttributes FAILED
    java.lang.AssertionError: expected:<225> but was:<0>
    	at org.junit.Assert.fail(Assert.java:89)
    	at org.junit.Assert.failNotEquals(Assert.java:835)
    	at org.junit.Assert.assertEquals(Assert.java:647)
    	at org.junit.Assert.assertEquals(Assert.java:633)
    	at com.liferay.document.library.internal.search.spi.model.index.contributor.test.DLFileEntryModelDocumentContributorTest._testFileEntryMetadataAttributesImageFileEntry(DLFileEntryModelDocumentContributorTest.java:443)
```

**Comment by Mikellorza** [comment]
Hi @AliciaGarciaGarcia ,

I made some changes related to your comments. Can you take a look again?

If you see that changes are fine we can run tests again to see if still fail but they should not fail.

thanks in advance!

**Review by jkappler — APPROVED** [review]

**Comment by jkappler** [comment]
ci:forward

**Pull request closed without merging**